Automattic/jetpack
# Fix comment action url to properly setup akismet on jp sites

The Akismet front-end ignores forms posting on other domains, which is the right thing to do.
For jp sites the comment form is iframed and they are posted to `https://jetpack.wordpress.com/jetpack-comment/`
In our frontend we replace the form action here: class.jetpack-renderer.php?r=10231ada&mo=40487&fi=1018#1020 but the akismet script is executed before that and the submit handler does not get attached.
This PR sets the form action properly so that it is later picked up by the akismet front-end.

Fixes #
127-gh-Automattic/verbum

## Proposed changes:

* Add function to get the comment action url
* Use this function to setup the default form action for comments

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR
* Post a comment on atomic site
* Inspect the comment post submit to see if the relevant akismet fields are posted.
<img width="1786" alt="Screenshot 2024-02-05 at 14 08 13" src="https://github.com/Automattic/jetpack/assets/7000684/4d6fad44-e4ce-4002-9322-6767003bf0c2">
